modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ResultsBar.js

```diff
@@ -121,6 +124,15 @@ const ResultsBar = ({
 					</ClayLink>
 				</ManagementToolbar.ResultsBarItem>
 			</ManagementToolbar.ResultsBar>
+
+			{title ? (
+				<ClayLayout.ContainerFluid
+					className={`c-mt-4 ${showTitle ? '' : 'sr-only'}`}
+					size="xl"
+				>
+					<h3>{title}</h3>
+				</ClayLayout.ContainerFluid>
+			) : null}
```

Comment:
To simplify, we don't need ternary expression here. Enough is something like `{!!title && (...)}` or `{Boolean(title) && (...)}`

Reply:
alright :smile: